engine/flink/model-util/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/aggregates.scala

```diff
@@ -48,6 +50,9 @@ object aggregates {
 
     override def zero: Number = null
 
+    override def isNeutralForAccumulator(element: Element, currentAggregate: Aggregate): Boolean =
+      addElement(element, currentAggregate) == currentAggregate
```

Comment:
maybe this should be the default implementation?

Reply:
Fixed